spec/request/in_progress_forms_request_spec.rb

```diff
@@ -18,7 +18,22 @@
     )
   end
 
-  describe 'GET' do
+  describe '#index' do
```

Comment:
index is GET /in_progress_forms
show is GET /in_progress_forms/:id
update is PUT/PATCH /in_progress_forms/:id